triples-generator/src/test/scala/io/renku/triplesgenerator/init/CliVersionCompatibilityVerifierImplSpec.scala

```diff
@@ -31,25 +30,21 @@ import scala.util.{Failure, Success, Try}
 class CliVersionCompatibilityVerifierImplSpec extends AnyWordSpec with MockFactory with should.Matchers {
 
   "run" should {
-    "return Unit if the cli version matches the cli version from the first pair in compatibility matrix" in {
-      val cliVersion = cliVersions.generateOne
-      val versionPairs = renkuVersionPairs.generateNonEmptyList() match {
-        case NonEmptyList(head, tail) => NonEmptyList(head.copy(cliVersion = cliVersion), tail)
-      }
-      val checker = new CliVersionCompatibilityVerifierImpl[Try](cliVersion, versionPairs)
+    "return Unit if the cli version matches the cli version from the first pair in compatibility config" in {
```

Comment:
I think the `first pair` in the presence of the new compatibility config does not hold anymore.